source/_docs/z-wave/controllers.markdown

```diff
@@ -63,6 +63,19 @@ The alternative to a stick is a hub that supports Z-Wave. Home Assistant support
 
 ## Controller Notes
 
+### 800 Series Controllers
+
+Some features are not supported with 800 series controllers yet.
+Unsupported
+ - Long Range
+ - NVM Backup/Restore 
```

Comment:
The HA integration does not support this anyways, even if the controller firmware does.

Reply:
Everything in this change was derived from [this GitHub issue](https://github.com/zwave-js/node-zwave-js/issues/5257). I figured it would be important to put all unsupported/supported features from that issue here because this documentation page isn't related to a specific integration. Z-wave JS UI does support these features, do I need to add a note for that somewhere?

Reply:
That list isn't ideal - I haven't looked at it in a while. Listing command classes specifically makes no sense.

If anything could be mentioned, it's the missing support for Long Range and/or NVM backup/restore, but I'll leave that up to @frenck to decide.

Reply:
Makes sense to me. I'll remove the supported list and just leave the unsupported features list.